src/YesSql.Core/Services/DefaultIdGenerator.cs

```diff
@@ -46,6 +46,9 @@ public Task InitializeAsync(IStore store, CancellationToken cancellationToken =
             return Task.CompletedTask;
         }
 
+        public Task InitializeAsync(IStore store)
+            => InitializeAsync(store, CancellationToken.None);
```

Comment:
I think it may be better to just add the default implementation in the interface instead so we can just remove them from one place in the future because I doubt anyone have a custom implementation

Reply:
I didn't suggest this because I am not sure this actually creates the method where it's missing. I tried on a random project to verify. But it might work as I didn't try with the full build, just looked at the generated code and IL.